boards/arm/nucleo_l4r5zi/nucleo_l4r5zi_defconfig

```diff
@@ -12,6 +13,14 @@ CONFIG_SOC_STM32L4R5XX=y
 # 80MHz system clock in 'normal power' mode
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=80000000
 
+CONFIG_STM32_LPTIM_TIMER=y
+CONFIG_STM32_LPTIM_CLOCK_LSE=y
+CONFIG_STM32_LPTIM_CLOCK_LSI=n
+# CONFIG_DEBUG=y
+CONFIG_TICKLESS_KERNEL=y
+CONFIG_TIMESLICING=n
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
```

Comment:
Do not impact default board configuration.
LPTIM conditional configuration could be done in board Kconfig.defconfig, but not here

Reply:
Enabling the LPTIMer is done in the board with 
CONFIG_STM32_LPTIM_TIMER=y